tsconfig.json

```diff
@@ -28,7 +28,7 @@
     "strict": true,
     "strictFunctionTypes": true,
     "strictNullChecks": true,
-    "strictPropertyInitialization": true,
+    "strictPropertyInitialization": false,
```

Comment:
@jrhender This options forces to append `undefined` to types of properties which can't be initialized in constructor (like `_ensRegisty`) and later always check them before using. It seems to me that these checking are not very useful, because they are just suppresses errors which indicates incorrect `IAM` initialization. Its a debatable question though. What do you think about it @dwojno ?

Reply:
Hmm, okay. In my opinion, it's good to be forced to append `undefined` if there is a possibility that it could be undefined. But if you guys both think it's unnecessary, then we can turn it off.